src/js/components/ResultItem.jsx

```diff
@@ -58,7 +58,7 @@ export class ResultItem extends Component {
     }
 
     toggleExpanded: Function = (e) => {
-        if (this.props.children) {
+        if (this.props.children && !getSelection().toString()) {
```

Comment:
`getSelection` might be undefined if we ever unit test this component in headless environment; you might want to do a simple check for it. Not critical though.